knowledge-base/grid-remove-aggregate-labels-in-excel-footer.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Remove the "Sum" and "Count" Strings from the Exported Excel File by the Grid
+description: Learn how to remove the "Sum" and "Count" strings from the exported Excel file by the Telerik Grid for Blazor. See how to add custom content to the footer cells in the generated Excel file.
+type: how-to
+page_title: How To Remove the "Sum" and "Count" Strings from the Exported Excel File by the Grid
+slug: grid-kb-remove-aggregate-labels-in-excel-footer
+tags: blazor, grid, treelist, excel, aggregates
+ticketid: 1676757, 1619584, 1610795
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor, <br /> TreeList for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to remove the **Sum:** and **Count:** strings in the Grid footer cells when using Excel export with `GridAggregates`?
+* How to get the customized text from the Grid `FooterTemplate` into the Excel download, instead of the aggregate function name itself?
+* How to make the exported Grid Excel file match the Grid footer template content?
+* How to remove the word **Sum** aggregate label in the footer of Grid when exporting data to Excel? I want to only keep the number in the footer of the Excel file.
```

Comment:
I suppose that this preserves the original wording from the customer. However the phrase "word Sum aggregate label" is confusing and clumsy. Maybe go with just "the Sum aggregate label" or "the word Sum from the aggregate labels".